content/news/XXXX-XX-XX-russians-war.md

```diff
@@ -0,0 +1,19 @@
+title: Russia's invasion of Ukraine
+authors: Daniel Schürmann
+tags: russian ukraine war invasion
+status: draft
+comments: yes
+
+Since 2022-02-24 we receive horrible news from Ukraine, which have become even worse since then. As an organization committed to freedom we feel the urge speak up and show solidarity for Ukraine.
+
+Russia's invasion of Ukraine cannot be justified by anything and has already been condemned by the UN as a violation of international law. The attack creates endless suffering for men, women and children in Ukraine but also for the many Russian soldiers who are sent into a senseless war that they do not want, do not understand and are not even allowed to call it as such.
+
+Mixxx and the FOSS movements respects users' freedom and stands also for freedom in other regards, including free speech and worldwide peaceful collaboration. Let's express this also in our DJ sets by integrating tracks from Ukrainian artist, Russian activists or anti-war tracks in general.
```

Comment:
```suggestion
Mixxx and the FOSS movement respect users' freedom and also stand for freedom in other regards, including free speech and worldwide peaceful collaboration. As an expression of solidarity, we suggest integrating tracks in our DJ sets from Ukrainian artists, Russian activists or by playing anti-war tracks in general.
```

Reply:
I think we should say a little more here, like:  "Although we are powerless to stop this war, and we are under no illusion that music alone will bring an end to the suffering, we feel it's important to speak out and make our voices heard."

Reply:
Yes that's a good idea. I have extended the sentence towards supporting all efforts helping Ukrainians.